examples/w3c_tracecontext_test/main.cc

```diff
@@ -0,0 +1,187 @@
+#include "opentelemetry/context/runtime_context.h"
+#include "opentelemetry/exporters/ostream/span_exporter.h"
+#include "opentelemetry/ext/http/client/curl/http_client_curl.h"
+#include "opentelemetry/ext/http/server/http_server.h"
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/propagation/http_trace_context.h"
+#include "opentelemetry/trace/provider.h"
+#include "opentelemetry/trace/scope.h"
+
+#include <algorithm>
+#include "nlohmann/json.hpp"
```

Comment:
Where / how do you install `nlohmann/json.hpp` for non-Windows OS?

Reply:
This relies on the `nlohmann-json` packages installed by our CI tools.